backend/src/database.py

```
@@ -33,7 +33,9 @@ def _load_index(self) -> Dict:
         except:
```

Ruff (0.13.1)

Do not use bare `except`

(E722)